sdks/go/pkg/beam/io/webapi/doc.go

```diff
@@ -0,0 +1,57 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Package webapi supports reading from or writing to Web APIs.
+
+Its design goals are to reduce the boilerplate of building Beam I/O connectors.
+The implementation is not yet complete (See tracking issue: https://github.com/apache/beam/issues/30423).
```

Comment:
Don't say it's incomplete. Simply tell the user to refer to the issue for the current status.

This avoids the very common case of the two sources of information (this doc comment, and the issue, mismatching).